identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -431,6 +431,7 @@ fido.proceed=Proceed
 fido.cancel=Cancel
 fido.submit=Submit
 fido.authenticator=Passkey
+provisioned.user.not.found=No associated local mapped user was found.
```

Comment:
Do we need the phrase "local mapped" here? IMO, it'd be too technical for an end user.

We sometimes ago removed a similar error message by replacing it with more user friendly error msg. Therefore, it'd be better if we can review this.

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/90354fddbe665b4b6574fe0a093710dade786d46